spec/Section 4 -- Composition.md

```diff
@@ -3179,6 +3179,127 @@ input BookFilter {
 In this invalid case, `title` is mandatory in Schema A but not defined in
 `Schema B`, causing inconsistency in required fields across schemas.
 
+#### Required Argument Missing
+
+**Error Code:**  
+
+`REQUIRED_ARGUMENT_MISSING_IN_SOME_SCHEMA`
```

Comment:
Could we try to keep the error codes <~35 characters?

Reply:
Maybe `REQUIRED_FIELD_ARGUMENT_MISSING`?

Reply:
FIELD_WITH_MISSING_REQUIRED_ARGUMENT